api/app/utils/emailer.py

```diff
@@ -25,13 +24,16 @@ def send_mail(recipient, subject, body_text='', body_html='', charset='UTF-8', m
     Raises:
         e -- [description]
     '''
+    sender_name = sender_name or SMTP_SENDER_NAME
+    sender_email = sender_email or SMTP_SENDER_EMAIL
+
```

Comment:
Is it possible that the email in the organisation table will differ from the one in the config? Because if so, won't that cause confusion if event level emails are sent from a different address?